bin/control_rods

```diff
@@ -82,21 +82,42 @@ main(["enable_firehose"]) ->
     halt(0);
 main(["disable_api"]) ->
     Node = connect(),
-    io:format("Disabling API (v2 & v3)...~n"),
-    Res = rpc:call(Node, logplex_api, set_status, [disabled]),
+    io:format("Disabling v3 API...~n"),
+    Res = rpc:call(Node, logplex_api_v3, set_status, [disabled]),
     io:format("Node: ~p~n", [Res]),
     disconnect(),
     halt(0);
 main(["enable_api"]) ->
     Node = connect(),
-    io:format("Enabling API (v2 & v3)...~n"),
-    Res = rpc:call(Node, logplex_api, set_status, [normal]),
+    io:format("Enabling v3 API ...~n"),
+    Res = rpc:call(Node, logplex_api_v3, set_status, [normal]),
     io:format("Node: ~p~n", [Res]),
     disconnect(),
     halt(0);
 main(["set_api_read_only"]) ->
     Node = connect(),
-    io:format("Setting API (v2 & v3) in read-only mode...~n"),
+    io:format("Setting v3 API in read-only mode...~n"),
+    Res = rpc:call(Node, logplex_api_v3, set_status, [read_only]),
+    io:format("Node: ~p~n", [Res]),
+    disconnect(),
+    halt(0);
+main(["disable_legacy_api"]) ->
+    Node = connect(),
+    io:format("Disabling legacy API ...~n"),
+    Res = rpc:call(Node, logplex_api, set_status, [disabled]),
```

Comment:
Should we call this `logplex_api_v2` so it's explicit?

Reply:
Nah.. we want to delete the module rather.